README.md

```diff
@@ -1,83 +1,135 @@
+| ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [![Build Status](https://dev.azure.com/osieng/engineering/_apis/build/status/product-readiness/OMF/osisoft.sample-omf-basic_api-python?repoName=osisoft%2Fsample-omf-basic_api-python&branchName=main&jobName=Tests_OCS)](https://dev.azure.com/osieng/engineering/_build/latest?definitionId=2637&repoName=osisoft%2Fsample-omf-basic_api-python&branchName=main) | [![Build Status](https://dev.azure.com/osieng/engineering/_apis/build/status/product-readiness/OMF/osisoft.sample-omf-basic_api-python?repoName=osisoft%2Fsample-omf-basic_api-python&branchName=main&jobName=Tests_EDS)](https://dev.azure.com/osieng/engineering/_build/latest?definitionId=2637&repoName=osisoft%2Fsample-omf-basic_api-python&branchName=main) | [![Build Status](https://dev.azure.com/osieng/engineering/_apis/build/status/product-readiness/OMF/osisoft.sample-omf-basic_api-python?repoName=osisoft%2Fsample-omf-basic_api-python&branchName=main&jobName=Tests_OnPrem)](https://dev.azure.com/osieng/engineering/_build/latest?definitionId=2637&repoName=osisoft%2Fsample-omf-basic_api-python&branchName=main) |
 
 The sample code in this topic demonstrates how to send OMF messages using Python.
 
 The samples were built and tested against Python 3. If you are using a different version you might encounter errors or unexepected behavior.
 
-To Run this Sample:
 
----
+## To Run this Sample:
 
 1. Clone the GitHub repository
 1. Install required modules: `pip install -r requirements.txt`
 1. Open the folder with your favorite IDE
-1. Rename the placeholder config file [config.placeholder.ini](config.placeholder.ini) to `config.ini`
-1. Update `config.ini` with your credentials
-1. Check and update the program to ensure you are sending to OCS or PI.
-1. Run `program.py` from commandline run `python program.py`
+1. Rename the placeholder config file [config-placeholder.json](config-placeholder.json) to config.json
```

Comment:
Typically, this would be `config.placeholder.json`; we use `.placeholder.` in the rest of our samples.

Reply:
I will changed to ```config.placeholder.json```